gatsby-site/src/components/Navbar/index.js

```diff
@@ -45,25 +47,48 @@ export default class ReactNavbar extends React.Component {
         <NavbarToggler onClick={this.toggle} />
           <Collapse isOpen={this.state.isOpen} navbar>
             <Nav className="ml-auto" navbar>
-              <NavItem>
-                <Scrollchor to="partners" className="nav-link">Partners</Scrollchor>
-              </NavItem>
-              <NavItem>
-                <Scrollchor to="#team" className="nav-link">Team</Scrollchor>
-              </NavItem>
-                <UncontrolledDropdown nav inNavbar>
-                  <DropdownToggle nav caret>
-                    Projects
-                  </DropdownToggle>
-                    <DropdownMenu right>
-                      <DropdownItem>
-                        <Scrollchor to="projects" className="nav-link">Phase Zero</Scrollchor>
-                      </DropdownItem>
-                      <DropdownItem>
-                        <Scrollchor to="projects" className="nav-link">Unstack</Scrollchor>
-                      </DropdownItem>
-                    </DropdownMenu>
-                </UncontrolledDropdown>
+              {this.state.isMainPage ?
+                <NavItem>
```

Comment:
Is there a simpler way to not repeat this conditional logic for each nav item? Maybe we could put it in a component that takes in an "isMainPage" prop with the 
"to", "className" and "children" prop?

Reply:
i'll try it